assets/src/block-templates/deep-dive-topic/block.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://schemas.wp.org/trunk/block.json",
+    "apiVersion": 2,
+    "name": "planet4-block-templates/deep-dive-topic",
+    "title": "Deep Dive Topic",
+    "category": "planet4-block-templates",
+    "icon": "editor-table",
+    "textdomain": "planet4-blocks-backend"
+  }
```

Comment:
I think this red symbol means that we are missing an empty line at the end of the file

Reply:
Fixed!